x-pack/plugins/security_solution/server/lib/detection_engine/notifications/schedule_notification_actions.ts

```diff
@@ -30,19 +34,33 @@ const convertToLegacyAlert = (alert: DetectionAlert) =>
     return acc;
   }, {});
 
+export const normalizeAlertForNotificationActions = (alert: DetectionAlert) => {
+  if (isThresholdRule(alert[ALERT_RULE_TYPE])) {
+    const { 'kibana.alert.threshold_result': thresholdResult, ...alertWithoutThresholdResult } =
```

Comment:
Can we use `[ALERT_THRESHOLD_RESULT]: thresholdResult` here?

Reply:
Good catch! Fixing in my next PR.